packages/inventory-compliance/src/SystemRulesTable.test.js

```diff
@@ -0,0 +1,157 @@
+import React from 'react';
+import { shallow, mount } from 'enzyme';
+import toJson from 'enzyme-to-json';
+import SystemRulesTable from './SystemRulesTable';
+import { IntlProvider } from 'react-intl';
+jest.mock('@redhat-cloud-services/frontend-components-remediations/remediationsApi');
+import * as remediationsApi from '@redhat-cloud-services/frontend-components-remediations/remediationsApi';
+import { SortByDirection } from '@patternfly/react-table';
+import { TITLE_COLUMN } from './Constants';
+
+const windowInsightsSpy = {
```

Comment:
Can you move this function to `/config/setUpTests`? So we don't have to mock it everytime?

Reply:
I tried by creating a file `config/setupJest.js`, and then changing package.lock with: 
```
    "jest": {
        "automock": false,
        "setupFiles": [
            "./setupJest.js"
        ]
    },
```

But it didn't quite work because we run the tests from the root of the monorepo, any ideas? So far I set `global.insights = global.window.insights;` in the global `config/setupTests.js` and that seems to be enough.

Reply:
Actually, the chrome config has been moved to global config so no need to define it here. And you can start using `window.insights` or as you mentioned setting `global.insights = global.window.insights;` should be enough.